### PR TITLE
Provider login and course admin area

### LINKED ISSUE #164: Provider login and course admin area
# Problem

Providers need to be able to log in and provide the system with updated course offerings.
# Proposed solution

Create a user area to manage course offerings.

### PR DESCRIPTION
**Pull request by meomancer**
Initializing for this work
@cchristelis 
it seems that we already have official

i think we could use it and link it to provider as an user of provider
and i think we could make page-like-admin to provide provider feature for handling courses

this fix #164

### CODE REVIEW

File: django_project/feti/models/address.py [hunk 1]
```diff
@@ -72,4 +72,4 @@ def address_line(self):
 
     class Meta:
         app_label = 'feti'
-        verbose_name_plural = 'Addreses'
```
Comment (cchristelis) [inline django_project/feti/models/address.py:75]: let's change this to to 'Addresses' rather than removing it?

File: django_project/feti/models/campus.py [hunk 2]
```diff
@@ -19,7 +19,7 @@
 class Campus(models.Model):
     """A campus where a set of courses are offered."""
     id = models.AutoField(primary_key=True)
-    campus = models.CharField('Provider', max_length=150, blank=True, null=True)
```
Comment (cchristelis) [inline django_project/feti/models/campus.py:22]: Why did you change this?

File: django_project/feti/models/campus.py [hunk 3]
```diff
@@ -50,7 +50,7 @@ class Campus(models.Model):
     class Meta:
         app_label = 'feti'
         ordering = ['campus']
-        verbose_name = 'Provider'
```
Comment (cchristelis) [inline django_project/feti/models/campus.py:53]: Let's leave this as provider. There was a big debate about this the last round, but it turns out `Provider` is the most correct name here.

**Comment by cchristelis** [comment]
@meomancer, this is a good start. Let's keep Campus as `Provider`.

File: django_project/user_profile/models/official.py [hunk 4]
```diff
@@ -13,3 +15,16 @@ class Official(models.Model):
     phone = models.CharField(
         validators=[phone_regex], blank=True, max_length=15
     )
+    provider = models.OneToOneField(Provider, related_name='official_provider')
```
Comment (meomancer) [inline django_project/user_profile/models/official.py]: @cchristelis 
as per discussion, we give just allowed form for provider
is my approach right? 

my idea is this user is representation of provider, for login and logout
and based on this user, we could give form for this provider

**Comment by meomancer** [comment]
@cchristelis 
I reverted models verbose name

updated :
- added profile page. In there i show what provider he is, and show list of course. In there, user can update course for that campus
- list just shown for his/her campus. if he doen't has provider, nothing will be shown

_note_:
- style is still minimalize, as well as text

screenshot from 2016-08-09 16 44 31
screenshot from 2016-08-09 16 44 42

**Comment by cchristelis** [comment]
@meomancer, while we are still busy with the project setup, let's merge this.
